nengo_gui/static/menu.js

```diff
@@ -27,6 +27,8 @@ Nengo.Menu.prototype.show = function (x, y, items) {
         return;
     }
 
+    this.menu_margin = 12;
```

Comment:
I think there are two problems here.  First, where is this magic number 12 coming from? What is it?  It's being used as a 12-pixel vertical offset inside the check_overflow code, and that is not clear from the variable naming here.  And instead of it being just a magic number, it should be computed from something so that it changes appropriately when the things that affect it change.

Second, why is this code out here rather than inside check_overflow, where it is used?  Why is it a member variable of the class, rather than just a temporary computed thing inside check_overflow?